crates/uv-resolver/src/prerelease.rs

```diff
@@ -84,6 +84,7 @@ impl PrereleaseStrategy {
 
                     if specifier
                         .iter()
+                        .filter(|spec| (spec.operator() != &Operator::NotEqual))
```

Comment:
@charliermarsh I just spot that there is `NotEqualStar` operator also defined. Should I add it here?  Is `flask!=2.0.0rc*` valid inequality?

Reply:
Yeah, I think you should add that here too.